tuber/__init__.py

```diff
@@ -12,6 +14,25 @@
     "setup.py",
 }
 
+DEFAULT_INCLUDES = ["*.py"]
+
+DEFAULT_EXCLUDES = [
```

Comment:
I'm not exactly sure how `black`'s default excludes works. On first glance it seems like it's a substring match which is why I added the asterisk to both sides of the string. I should also add a permalink to black here.